crates/dips/src/lib.rs

```diff
@@ -18,6 +20,7 @@ pub mod ipfs;
 pub mod price;
 #[cfg(feature = "rpc")]
 pub mod proto;
+pub mod registry;
```

Comment:
If everything within this file is only for testing purposes, mark it as test only.

```suggestion
#[cfg(test)]
pub mod registry;
```